vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -638,6 +638,15 @@ public long getPlayProgressMillis() {
     // <editor-fold desc="VOD">
 
     /**
+     * @return the {@link SvodInteraction}. Will be null if this video is not part of SVOD
+     */
+    @Nullable
+    public SvodInteraction getSvodInteraction() {
+        InteractionCollection interactions = mMetadata.getInteractions();
```

Comment:
Do you think we should check the metadata object for nullity, or can we safely assume it is not null? It's not marked as either nonnull or nullable, and it seems that we sometimes check to see if it's null and sometimes don't in the code.